sale_order_customer_free_ref/models/sale_order.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    _client_order_ref_separator = " - "
+
+    client_order_ref = fields.Char(
+        compute="_compute_client_order_ref",
+        string="Customer Reference",
+        store=True,
+        copy=False,
+        readonly=False,
```

Comment:
By doing that, no need for an `inverse`? What happens if we write a value in `client_order_ref`?

Reply:
Maybe the failing job answers my question: can you check?

Reply:
The last fixup adds an inverse method...